scripts/deploy/src/cluster/get_output.sh

```diff
@@ -125,7 +125,7 @@ fi
 # fail if mandetory flags aren't set
 if [[ "$resourceGroup" == "" ]]; then fatal "--resource-group flag must be defined"; fi
 if [[ "${ARG_FILE}" == "" && ( ! -e "${ARG_FILE}")]]; then fatal "--argfile flag must be defined with a valid json argument file [${ARG_FILE}]"; fi
-# fail of mandetory JSON fields in the --argfile aren't set
+# fail of mandetory JSON fields in the --argfile are not set
```

Comment:
If you're going to change this then also s/mandetory/mandatory (multiple occurrences in the file) ;)

Reply:
Hi @stuartpreston.  I've done the typo fix in multiple files.  

for FILE in $(ack "mandetory" -l); do sed -i '' 's/mandetory/mandatory/g' $FILE; done

:)